ci-operator/config/redhat-performance/jetlag/redhat-performance-jetlag-main.yaml

```diff
@@ -46,6 +46,22 @@ tests:
     env:
       NUM_WORKER_NODES: "0"
     workflow: openshift-qe-installer-bm-deploy
+- always_run: false
+  as: deploy-mno-hybrid
+  capabilities:
+  - intranet
+  restrict_network_access: false
+  steps:
+    cluster_profile: metal-perscale-cpt
+    env:
+      FOREMAN_OS: RHEL 9.4
+      NUM_HYBRID_WORKER_NODES: "2"
```

Comment:
If you only have one hypervisor available shouldn't the value of NUM_HYBRID_WORKER_NODES be 1?

Reply:
Also, please move this test out of this file into: https://github.com/openshift/release/blob/master/ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__metal-4.19-nightly-x86.yaml

Reply:
The `NUM_HYBRID_WORKER_NODES` is the amount of VMs created for worker nodes so one hypervisor could handle multiple VMs.